rep-0127.rst

```diff
@@ -240,6 +240,17 @@ alphabetic, numeric or underscore characters [1]_.  The package name
 should be unique within the ROS community.  It may differ from the
 folder name into which it is checked out, but that is *not* recommended.
 
+The following recommended exemptions apply, which are optional for
+implementations:
+
+- For packages which have a ``build_tool`` dependency on a package other
+  than ``catkin``, dashes are permitted. This is to support maintaining
```

Comment:
Sure, if desired. I hadn't seen anyone else weigh in on that thread, so I wasn't sure what the consensus opinion was.